oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/ModelInstaller.java

```diff
@@ -58,6 +57,9 @@ public void whenCreating(Model model) throws StorageException {
         }
     }
 
+    public void start() {
```

Comment:
What does `ModelInstaller#start` mean?

Reply:
> What does `ModelInstaller#start` mean?

This is to provide a hook for child-implementations of model installer to do initializations, for now, we do column overrides (e.g. `overrideColumnName("value", "value_");`) in this method.

Previously we do `overrideColumnName("value", "value_");` in the constructor of model installer, this is not an appropriate phase because `overrideColumnName("value", "value_");` requires services that must be already registered to module manager and it might raise `Still in preparing stage` exception if the construction of model installer is too early.